### PR TITLE
Updated P20 ADCP cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Updated the PM cruise-specific cal csvs per latest info from Sheri.  
Revised Table 5-1 from the cal csv procedure is attached with the updated parameter highlighted yellow.  

[3408-00201_Table 4-1.docx](https://github.com/WHOIGit/ooicgsn-asset-management/files/15067041/3408-00201_Table.4-1.docx)

### CODE REVIEW

File: calibration/ADCPTG/CGINS-ADCPTG-24543__20240401.csv [hunk 1]
```diff
@@ -4,5 +4,5 @@ serial,name,value,notes
 24543,CC_scale_factor3,0.45,constant
 24543,CC_scale_factor4,0.45,constant
 24543,CC_bin_size,400,in cm
-24543,CC_dist_first_bin,901,in cm
+24543,CC_dist_first_bin,840,in cm
```
Comment (cooleyky) [inline calibration/ADCPTG/CGINS-ADCPTG-24543__20240401.csv:7]: looks good

File: calibration/ADCPTG/CGINS-ADCPTG-19151__20240401.csv [hunk 2]
```diff
@@ -4,5 +4,5 @@ serial,name,value,notes
 19151,CC_scale_factor3,0.45,constant
 19151,CC_scale_factor4,0.45,constant
 19151,CC_bin_size,400,in cm
-19151,CC_dist_first_bin,901,in cm
+19151,CC_dist_first_bin,840,in cm
```
Comment (cooleyky) [inline calibration/ADCPTG/CGINS-ADCPTG-19151__20240401.csv:7]: looks good

File: calibration/ADCPTG/CGINS-ADCPTG-24544__20240401.csv [hunk 3]
```diff
@@ -4,5 +4,5 @@ serial,name,value,notes
 24544,CC_scale_factor3,0.45,constant
 24544,CC_scale_factor4,0.45,constant
 24544,CC_bin_size,400,in cm
-24544,CC_dist_first_bin,901,in cm
+24544,CC_dist_first_bin,840,in cm
```
Comment (cooleyky) [inline calibration/ADCPTG/CGINS-ADCPTG-24544__20240401.csv:7]: looks good

File: calibration/ADCPSL/CGINS-ADCPSL-18445__20240401.csv [hunk 4]
```diff
@@ -4,5 +4,5 @@ serial,name,value,notes
 18445,CC_scale_factor3,0.45,constant
 18445,CC_scale_factor4,0.45,constant
 18445,CC_bin_size,800,in cm
-18445,CC_dist_first_bin,1695,in cm
+18445,CC_dist_first_bin,1662,in cm
```
Comment (cooleyky) [inline calibration/ADCPSL/CGINS-ADCPSL-18445__20240401.csv:7]: looks good

File: calibration/ADCPSL/CGINS-ADCPSL-21497__20240401.csv [hunk 5]
```diff
@@ -4,5 +4,5 @@ serial,name,value,notes
 21497,CC_scale_factor3,0.45,constant
 21497,CC_scale_factor4,0.45,constant
 21497,CC_bin_size,800,in cm
-21497,CC_dist_first_bin,1695,in cm
+21497,CC_dist_first_bin,1662,in cm
```
Comment (cooleyky) [inline calibration/ADCPSL/CGINS-ADCPSL-21497__20240401.csv:7]: looks good

**Review by cooleyky — APPROVED** [review]
Updates look good

File: calibration/ADCPSL/CGINS-ADCPSL-21497__20240401.csv
Comment (reedan88) [inline calibration/ADCPSL/CGINS-ADCPSL-21497__20240401.csv:1]: Looks good

File: calibration/ADCPSL/CGINS-ADCPSL-18445__20240401.csv
Comment (reedan88) [inline calibration/ADCPSL/CGINS-ADCPSL-18445__20240401.csv:1]: Looks good.

File: calibration/ADCPTG/CGINS-ADCPTG-24543__20240401.csv
Comment (reedan88) [inline calibration/ADCPTG/CGINS-ADCPTG-24543__20240401.csv:1]: Looks good

File: calibration/ADCPTG/CGINS-ADCPTG-19151__20240401.csv
Comment (reedan88) [inline calibration/ADCPTG/CGINS-ADCPTG-19151__20240401.csv:1]: Looks good.

File: calibration/ADCPTG/CGINS-ADCPTG-24544__20240401.csv
Comment (reedan88) [inline calibration/ADCPTG/CGINS-ADCPTG-24544__20240401.csv:1]: Looks good.

**Review by reedan88 — APPROVED** [review]

**Pull request merged**